.circleci/config.yml

```diff
@@ -158,6 +158,11 @@ jobs:
           name: Deploy static files to Google Cloud Storage Bucket
           command: |
             gsutil -m cp -r -z "js,css,map,txt" ./static/** "gs://${GOOGLE_CLOUD_BUCKET}/${CIRCLE_TAG/#v}/"
+      - run:
+          name: Deploy introspection json to Google Cloud Storage Bucket
+          command: |
+            gsutil cp "src/core/server/graph/schema/__generated__/introspection.json" "gs://${GOOGLE_CLOUD_BUCKET}"
+            gsutil cp "src/core/server/graph/schema/__generated__/introspection.json" "gs://${GOOGLE_CLOUD_BUCKET}/${CIRCLE_TAG/#v}/"
```

Comment:
These files are not available here without copying them out of the container in the `Copy static assets from docker container` step. Ensure you copy the files in that step.